### PR TITLE
Allow deleting poll options #599

### PR DESCRIPTION
**Pull request by jszanto**

### CODE REVIEW

File: module/Frontpage/view/frontpage/poll/request.phtml [hunk 1]
```diff
@@ -100,11 +101,11 @@ ob_start();
                     <?= $this->formElementErrors($element) ?>
                 </div>
             </div>
-    <?php endforeach ?>
+    <?php $i++; endforeach ?>
```
Comment (kokx) [inline module/Frontpage/view/frontpage/poll/request.phtml]: This looks a little strange to me. But don't really know a quick way to fix it.

Comment (jszanto) [inline module/Frontpage/view/frontpage/poll/request.phtml]: I could put the `$i++` on a seperate line if your eyes like that better.

Comment (kokx) [inline module/Frontpage/view/frontpage/poll/request.phtml]: I guess they would.

**Comment by jszanto** [comment]
@kokx moved it

**Pull request merged**